test/e2e/framework/framework.go

```diff
@@ -244,49 +208,54 @@ func RunConfigResolver(t *testing.T, opt ConfigResolverOptions) string {
 	})...)
 }
 
-func runAccessory(t *testing.T, command string, args ...string) string {
-	t.Helper()
-	if _, err := exec.LookPath(command); err != nil {
-		t.Fatalf("no `%s` binary was found: %v", command, err)
-	}
+func runAccessory(binder func(port string, cmd CiOperatorCommand), command string, args ...string) AccessoryFunc {
```

Comment:
if `command` was a `func(port, healthport int) string` you wouldn't need to require all commands to have identically named flags for port/health-port